src/Rules/VideoIdRule.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Vimeo\Laravel\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+use Vimeo\Laravel\Facades\Vimeo;
+
+class VideoIdRule implements Rule
```

Comment:
This is already separately in this PR https://github.com/vimeo/laravel/pull/92/files